config/operators/che/che_cluster.yaml

```diff
@@ -12,9 +12,9 @@ spec:
   server:
     customCheProperties:
       CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT: <username>-code
-      CHE_WORKSPACE_DEFAULT__CPU__LIMIT__CORES: '0.2'
+      CHE_WORKSPACE_DEFAULT__CPU__LIMIT__CORES: '0.4'
       CHE_WORKSPACE_DEFAULT__CPU__REQUEST__CORES: '0.03'
-      CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__LIMIT__CORES: '0.2'
+      CHE_WORKSPACE_SIDECAR_DEFAULT__CPU__LIMIT__CORES: '0.4'
```

Comment:
This is not related to autoscaler but I was too lazy to create another PR for that :)
It makes Che workspace/IDE a bit faster.